instaclustr/resource_kafka_user.go

```diff
@@ -43,6 +44,21 @@ func resourceKafkaUser() *schema.Resource {
 				Default:  "none",
 				ForceNew: true,
 			},
+
+			"options": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"sasl-scram-mechanism": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
```

Comment:
**[Question]** I'm curious about the use of `ForceNew: true` for both the options schema and the SCRAM field. Is there a reason that both have to be re-created? My gut feeling is that it accounts for future options settings, but still seems overkill.

Reply:
Addressed